### PR TITLE
Bug 1363467 - Toggle classifications

### PR DESCRIPTION
**Pull request by camd**
* Toggle to show / hide infra and intermittent tests.
* Show stars on any classified tests, not just infra and intermittent
* Show only Tier 1 and 2 jobs.  Skip Tier 3
* Show a Tier badge on Tier 2 jobs

### CODE REVIEW

File: src/redux/modules/groups.js [hunk 1]
```diff
@@ -130,6 +142,7 @@ const initialState = {
   groups: {},
   rowData: {},
   counts: { failed: 0, intermittent: 0, infra: 0, success: 0, running: 0, pending: 0 },
+  hideClassified: {'infra': true, 'intermittent': true},
```
Comment (eliperelman) [inline src/redux/modules/groups.js]: Remove single quotes from these object keys. Insert a space inside the braces for object padding, consistent with other objects.

File: src/ui/StatusNavbar.js [hunk 2]
```diff
@@ -2,56 +2,86 @@ import React from 'react';
...
-      <Badge color="info">{props.counts.running} Running Jobs</Badge>
-    </span>
-
-    <span className="navbar-text">
-      <Badge>{props.counts.pending} Pending Jobs</Badge>
-    </span>
-  </Navbar>
-);
+class StatusNavbar extends React.Component {
+
```
Comment (eliperelman) [inline src/ui/StatusNavbar.js]: Remove extra line.

File: src/ui/StatusNavbar.js [hunk 3]
```diff
@@ -2,56 +2,86 @@ import React from 'react';
...
+          </span>
+        </Nav>
+
+        <span className="navbar-text">
+          <Badge color="danger">{this.props.counts.failed} Other Failed Tests</Badge>
+        </span>
+
+        <span className="navbar-text toggle-count" onClick={() => this.toggleHideClassified('infra')}>
+          <Badge color="infra">
+            <Icon name={this.props.hideClassified['infra'] ? "square-o" : "check-square-o"}/>
```
Comment (eliperelman) [inline src/ui/StatusNavbar.js]: Use single quotes on JS; double quotes is for JSX attributes only (and this is interpolated JS).

File: src/ui/StatusNavbar.js [hunk 4]
```diff
@@ -2,56 +2,86 @@ import React from 'react';
...
+            {this.props.counts.infra} Infra Tests
+          </Badge>
+        </span>
+
+        <span className="navbar-text toggle-count" onClick={() => this.toggleHideClassified('intermittent')}>
+          <Badge color="intermittent">
+            <Icon name={this.props.hideClassified['intermittent'] ? "square-o" : "check-square-o"}/>
```
Comment (eliperelman) [inline src/ui/StatusNavbar.js]: Replace double quotes with single quotes.

Comment (camd) [inline src/ui/StatusNavbar.js]: Dang me and my double-quote madness!!  ;)

**Comment by camd** [comment]
Thanks for the review, man.  Fixes applied in a fixup.

**Pull request merged**